apps/admin-gui/src/app/shared/components/dialogs/add-member-dialog/add-member-dialog.component.html

```diff
@@ -1,80 +1,91 @@
-    </div>
-    <div class="mt-3" *ngIf="!!members && !loading">
-      <app-members-candidates-list
-        #list
-        [group]="data.group"
-        [tableId]="tableId"
-        [members]="members"
-        [selection]="selection"
-        [type]="data.type"
-        [blockManualAdding]="manualAddingBlocked"
-        *ngIf="firstSearchDone">
-      </app-members-candidates-list>
+      <div class="no-bounce-scrollbar">
+        <mat-spinner *ngIf="members === null" class="mr-auto ml-auto"></mat-spinner>
+      </div>
+      <div class="mt-3" *ngIf="!!members && !loading">
+        <app-members-candidates-list
+          [tableId]="tableId"
+          [members]="members"
+          [selection]="selection"
+          [blockManualAdding]="manualAddingBlocked">
+        </app-members-candidates-list>
+      </div>
     </div>
-  </div>
-  <mat-spinner class="mr-auto ml-auto" *ngIf="processing"></mat-spinner>
-  <div mat-dialog-actions *ngIf="!processing">
-    <button mat-flat-button class="ml-auto" (click)="onCancel()">
-      {{'DIALOGS.ADD_MEMBERS.CANCEL' | translate}}
-    </button>
-    <span
-      matTooltip="{{'DIALOGS.ADD_MEMBERS.ADD_MEMBER_PERMISSION_TOOLTIP' | translate}}"
-      matTooltipPosition="above"
-      [matTooltipDisabled]="selection.selected.length === 0 || this.data.type !== 'group' || (list !== undefined && list.addAuth)">
+    <div mat-dialog-actions>
+      <button mat-flat-button class="ml-auto" (click)="cancel.emit()">
+        {{'DIALOGS.ADD_MEMBERS.CANCEL' | translate}}
+      </button>
+      <span
+        matTooltip="{{'DIALOGS.ADD_MEMBERS.ADD_MEMBER_PERMISSION_TOOLTIP' | translate}}"
+        matTooltipPosition="above"
+        [matTooltipDisabled]="selection.selected.length === 0 || addAuth">
+        <button
+          *ngIf="inviteAuth"
+          [disabled]="selection.selected.length === 0"
+          [matMenuTriggerFor]="menu"
+          class="ml-2 dropdown-toggle"
+          color="accent"
+          mat-flat-button>
+          {{'DIALOGS.ADD_MEMBERS.INVITE' | translate}}
+        </button>
+        <mat-menu #menu="matMenu">
+          <button (click)="invite.emit(lang)" *ngFor="let lang of languages" mat-menu-item>
+            {{'DIALOGS.ADD_MEMBERS.INVITE_IN_LANGUAGE' | translate}}
+            {{'SHARED_LIB.LANGUAGES.'+lang | uppercase | translate}}
+            {{'DIALOGS.ADD_MEMBERS.LANGUAGE' | translate}}
+          </button>
+        </mat-menu>
+      </span>
       <button
-        *ngIf="this.data.type !== 'group' || (this.data.type === 'group' && inviteAuth)"
-        [disabled]="selection.selected.length === 0 || loading"
-        [matMenuTriggerFor]="menu"
-        class="ml-2 dropdown-toggle"
+        mat-flat-button
+        class="ml-2"
         color="accent"
-        mat-flat-button>
-        {{'DIALOGS.ADD_MEMBERS.INVITE' | translate}}
+        data-cy="add-button"
+        (click)="add.emit()"
+        [disabled]="selection.selected.length === 0 || !addAuth">
```

Comment:
maybe we would like some sort of tooltip when this button is disable due to priviledges